2-repositories/iglu-server/project/BuildSettings.scala

```diff
@@ -78,18 +31,21 @@ object BuildSettings {
     Cmd("RUN", "apt install -y postgresql-client-9.6")
   )
 
+  lazy val javaSettings = Seq(
```

Comment:
Do we still need these. SBT complains they have no effect.

Reply:
Shouldn't be necessary since the issue with Scala stack overflowing was fixed, I'll remove those.